src/UIKit/UIGraphics.cs

```diff
@@ -125,10 +130,16 @@ public static void BeginPDFContext (string file, CGRect bounds, CGPDFInfo docume
 		[DllImport (Constants.UIKitLibrary)]
 		extern static void UIGraphicsEndPDFContext ();
 
-		public static void EndPDFContent () {
+		public static void EndPDFContext () {
 			UIGraphicsEndPDFContext ();
 		}
 
+#if !XAMCORE_5_0
+		[EditorBrowsable (EditorBrowsableState.Advanced)]
```

Comment:
I think you can be more secretive:

```suggestion
		[EditorBrowsable (EditorBrowsableState.Never)]
```